### PR TITLE
Making Downloads Trackable (see #1141)

### PR DESCRIPTION
**Pull request by fhemberger**
See #1141 for discussion.

### CODE REVIEW

File: layouts/partials/footer.hbs [hunk 1]
```diff
@@ -58,7 +58,21 @@
     s=o.getElementsByTagName(e)[0];j.async=1;j.src='//www.google-analytics.com/analytics.js';
     s.parentNode.insertBefore(j,s)}(window,document,'ga','script');
 
+    if (!ga) return;
```
Comment (lpinca) [inline layouts/partials/footer.hbs:61]: Isn't this redundant? The above IIFE should always add the global `ga` no?

File: locale/en/foundation/case-studies.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ preferences (current and future) for front end, back end and IoT developers.
 
 * [Read the full announcement.](/en/blog/announcements/nodejs-foundation-survey/)
 * [See the infographic.](/static/documents/2016-survey-infographic.png)
-* [Read the full report.](/static/documents/2016-survey-report.pdf)
+* <a href="/static/documents/2016-survey-report.pdf" data-casestudy="Node.js 2016 Report">Read the full report.</a>
```
Comment (lpinca) [inline locale/en/foundation/case-studies.md:19]: Is this a case of study?

Comment (fhemberger) [reply locale/en/foundation/case-studies.md:19]: It was treated that way in the original PR. ¯\_(ツ)_/¯

File: layouts/partials/footer.hbs (hunk 1 above)
Comment (fhemberger) [reply layouts/partials/footer.hbs:61]: If you use an ad-blocker, the GA script will not be loaded and thus the `ga` object doesn't exist.

**Review by lpinca — APPROVED** [review]

**Comment by phillipj** [comment]
Just tested this on nodejs.org and doesn't seem to work, at least not in Chrome on my machine. There is no request sent to Google when clicking these links.

I assume that is primarily caused by this error:

screenshot1

I'll have a look and open a PR shortly.

**Comment by mikeal** [comment]
This is great, thanks for cleaning this up :+1:

**Comment by lpinca** [comment]
Ops, that `return` seems to be outside a function so it should be probably be changed to `if (ga) { ... }`. Sorry missed that during review.

**Pull request merged**